source/guides/core-concepts/variables-and-aliases.md

```diff
@@ -260,7 +259,7 @@ describe('parent', function () {
 
 The most common use case for sharing context is when dealing with {% url `cy.fixture()` fixture %}.
 
-Often times you may load in a fixture in a `beforeEach` hook but want to utilize the values in your tests.
+Often times you may load a fixture in a `beforeEach` hook but want to utilize the values in your tests.
```

Comment:
The double occurrence of the word _"in"_ tripped by tongue so I elided one of 'em: there's no loss of clarity.